api/cluster.capnp

```diff
@@ -0,0 +1,27 @@
+using Go = import "/go.capnp";
+
+@0xfcf6ac08e448a6ac;
+
+$Go.package("cluster");
+$Go.import("github.com/wetware/ww/internal/api/cluster");
+
+
+
+struct Record {
+    peer @0 :Text;
+    ttl @1 :Int64;
+    seq @2 :UInt64;
+}
+
+struct Iteration {
+    record @0 :Record;
+    dedadline @1 :Int64;
+}
+
+interface Cluster {
```

Comment:
We said we were going to rename to `Cluster`. In the end, it's the `View` of a `Cluster`. So, as we don't have any other interface or service related to the `Cluster` package on `casm`, I would stick to `Cluster` instead of `View`, as `View` is very generic.

Reply:
Ah, ok.  I misunderstood:  I thought we were renaming the package.  I'm good with this. Marking as resolved.